demo/README.md

```diff
@@ -28,6 +28,15 @@ docker run --rm -it inputoutput/hydra:hydra-tui-latest --connect localhost:4002
 docker run --rm -it inputoutput/hydra:hydra-tui-latest --connect localhost:4003 # carol's hydra-node
 ```
 
+NOTE: You can query the `cardano-node` using the host-mounted socket file in
+`devnet/ipc/node.socket` (requires write permissions), e.g.
+
+``` sh
+sudo chmod a+w devnet/ipc/node.socket
```

Comment:
Not quite convinced about this and why this is necessary. Running docker as root is probably not a good idea, and as a consequence, requires us to do things like this. -> Red bin.

Reply:
I agree. You mean running the docker container with `-u $UID`, right? Not sure why I haven't questioned this myself.